app/views/admin/users/_form.html.erb

```diff
@@ -22,6 +22,11 @@
     <div class="col-sm-10"><%= f.text_field :username %></div>
   </div>
 
+  <div class="field-wrapper">
+    <%= f.label :role, class: 'col-sm-2 control-label' %>
+    <div class="col-sm-10"><%= f.select :role, I18n.t('users.roles').invert %></div>
```

Comment:
I think that its not a good idea to build this select using i18n translation, would be safer to centralize this values in the model, and use i18n to just translate them.

Also, I think that we can drop support for some languages, as we are mostly using google translator during the development. We should find some people, or a better tool to support them.